tests/e2e/tests/test_base.py

```diff
@@ -77,12 +76,7 @@ def request_with_headers(self, url, params, user_agent=_user_agent_firefox, loca
         headers = {'user-agent': user_agent,
                    'accept-language': locale,
                    'Connection': 'close'}
-        try:
-            response = requests.head(url, headers=headers, verify=True, timeout=15, params=params, allow_redirects=True)
-        except requests.RequestException as e:
-            request_url = '%s/?%s' % (url, urlencode(params))
-            raise AssertionError('Failing URL: %s redirected to %s Error message: %s' % (request_url, response.url, e))
-        return response
+        return requests.head(url, headers=headers, verify=True, timeout=15, params=params, allow_redirects=True)
```

Comment:
+1, apologies for missing the try/except problem. I knew this but didn'l look closely at the previous pr closely enough.

Reply:
It's okay, this pre-dates the previous pull request and was introduced in f5782f0b13300f40792fdc38e5a7b74d18d636b0.